rasa/core/brokers/kafka.py

```diff
@@ -75,7 +77,12 @@ def __init__(
         logging.getLogger("kafka").setLevel(loglevel)
 
     @classmethod
-    def from_endpoint_config(cls, broker_config) -> Optional["KafkaEventBroker"]:
+    async def from_endpoint_config(
```

Comment:
we could technically do it without that by using `loop.run_until_complete` 🤔 Currently it wouldn't make a difference but it might be nicer in the future if we wanna do something else in between